### PR TITLE
Companies Logos

### PR DESCRIPTION
**Pull request by SaraVieira**
Creates a /users and adds some users to the hompage too

### CODE REVIEW

**Comment by SaraVieira** [comment]
You can see here: http://pr1552.cs.lbogdan.tk/users

And in the bottom of the homepage

File: packages/homepage/src/screens/home/Users/index.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+              logoURL
+              link
+              pinned
+            }
+          }
+        }
+      }
+    `}
+    render={({ allUsersJson: { edges } }) => (
+      <Container>
```
[the remark this answers is not in the document]
Comment (CompuIves) [reply packages/homepage/src/screens/home/Users/index.js]: On the homepage this looks fine already.

File: packages/homepage/src/pages/users.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        </Heading2>
+        <p
+          css={`
+            text-align: center;
+            color: white;
+          `}
+        >
+          Some amazing companies and projects using CodeSandbox
+        </p>
+        <Grid>
```
Comment (CompuIves) [inline packages/homepage/src/pages/users.js]: Could we extract this component so we can reuse it on both pages?

File: packages/homepage/src/pages/users.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (SaraVieira) [reply packages/homepage/src/pages/users.js]: Changed to who-uses-codesandbox :)

File: packages/homepage/src/screens/home/Users/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (SaraVieira) [reply packages/homepage/src/screens/home/Users/index.js]: Added 2rem :D

File: packages/homepage/src/utils/shuffleArray.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+function shuffleArray(array) {
```
[the remark this answers is not in the document]
Comment (SaraVieira) [reply packages/homepage/src/utils/shuffleArray.js]: Danke

File: packages/homepage/src/pages/users.js (hunk 2 above)
Comment (SaraVieira) [reply packages/homepage/src/pages/users.js]: Created a `Companies.js` component

**Comment by SaraVieira** [comment]
Closes https://github.com/CompuIves/codesandbox-client/issues/783

**Pull request merged**